script/Arcueid/Day 7/07_00_ARC07_1D.txt

```diff
@@ -172,32 +173,35 @@ The apartment was the same as before, but Arcueid was nowhere to be found.
 [94bb40b12623a56c752f8e5cb353d5d9acf5bac1]{
 -- Page 123, Offset 16885.
 -- 少なくともあいつの痕跡はここにある。煙のように何もかも消えた訳ではない事に、ほっと胸を撫で下ろした。
-At the very least, I can feel traces of her in this room. I feel a sense of relief knowing that everything hadn't gone up in smoke.
+// I think "gone up in smoke" implies something else, though it still fits.
+At the very least, the traces of her presence are still in this room. I feel a sense of relief knowing that everything hasn't vanished like smoke.
 }
 [d09cc04177c2d9c6a1b03e18b006ef2bfe32cdbd]{
 -- Page 124, Offset 16886.
 -- 「いいさ、そう簡単に捕まえられるなんて思ってない。
-"Listen up. Finding her is not going to be that easy.
+"It's fine. Finding her isn't going to be that easy.
```

Comment:
literally it's "I don't think finding her is going to be so easy" but it makes a lot more sense in past.
> It's fine. I didn't think finding her was going to be so easy.